packages/react-email-templates/package.json

```diff
@@ -0,0 +1,23 @@
+{
+	"name": "emails",
+	"version": "0.0.18",
+	"private": true,
+	"workspaces": [
+		".react-email"
```

Comment:
what's this for?

Reply:
this was how it was after the automatic setup https://react.email/docs/getting-started/automatic-setup - there's a .react-email subfolder but everything seems to work ok even if I delete this :)